Content.Client/Guidebook/Controls/GuideReagentEmbed.xaml.cs

```diff
@@ -128,8 +128,16 @@ private void GenerateControl(ReagentPrototype reagent)
                     continue;
 
                 var groupLabel = new RichTextLabel();
-                groupLabel.SetMarkup(Loc.GetString("guidebook-reagent-effects-metabolism-group-rate",
-                    ("group", _prototype.Index<MetabolismGroupPrototype>(group).LocalizedName), ("rate", effect.MetabolismRate)));
+                if (effect.MetabolismRate >= 0)
+                {
+                    groupLabel.SetMarkup(Loc.GetString("guidebook-reagent-effects-metabolism-group-rate",
+                        ("group", _prototype.Index<MetabolismGroupPrototype>(group).LocalizedName), ("rate", effect.MetabolismRate)));
+                }
+                else
+                {
+                    groupLabel.SetMarkup(Loc.GetString("guidebook-reagent-effects-metabolism-group-norate",
+                        ("group", _prototype.Index<MetabolismGroupPrototype>(group).LocalizedName)));
+                }
```

Comment:
Please just remove this code. metabolism groups without a rate don't exist now that you don't use this code pathway.

Reply:
Fair enough.